plugins/CoreHome/angularjs/comparisons/comparisons.component.html

```diff
@@ -4,8 +4,8 @@ <h3>{{ 'General_Comparisons'|translate }}</h3>
     <div class="comparison card" ng-repeat="comparison in $ctrl.comparisonsService.getSegmentComparisons() track by $index">
         <div class="comparison-type">{{ 'General_Segment'|translate }}</div>
 
-        <div class="title" title="{{ comparison.title }}">
-            {{ comparison.title }}
+        <div class="title" title="{{ comparison.title }}&lt;br/&gt;{{ comparison.params.segment }}">
```

Comment:
not sure, but does it maybe need some escaping for XSS, as a segment definition can contain any string?

Reply:
angularjs does this by default, so should be ok.

Reply:
@diosmosis this is what it looks for me

![image](https://user-images.githubusercontent.com/273120/73143810-65fce000-4103-11ea-87e7-3ecce2641f11.png)

with this segment

![image](https://user-images.githubusercontent.com/273120/73143814-6c8b5780-4103-11ea-9df5-7adba62554d0.png)
in the DB the segment is stored as `pageUrl=@b`